build.sbt

```diff
@@ -7,7 +7,7 @@ val RootSettings =
 
 val CommonsSettings =
   Seq(
-    version := Option(System.getProperty("version")).getOrElse("SNAPSHOT"),
+    version := Option(System.getProperty("version")).getOrElse("0.1-SNAPSHOT"),
```

Comment:
the snapshot version needs to look like a semver afait... when it doesn't the `sbt publishLocal` complains that a previous version exists after you repeated a publication and ask you to delete the previously published artifact from the ivy2 directory... it would override previously published artifacts if it can tell that is indeed a snapshot version